Documentation/guides/building-apps/build-properties.md

```diff
@@ -485,6 +485,23 @@ Support for this property was added in Xamarin.Android 9.4.
 
 This property is `False` by default.
 
+
+## AndroidEnableRestrictToAttributes
+
+A boolean property that determines whether types and members marked with
+the Java annotation `androidx.annotation.RestrictTo` will be marked with an
+`[Obsolete]` attribute in the C# binding.
+
+This `[Obsolete]` attribute has a descriptive message explaining that the
+Java package owner considers the API to be "internal" and warns against its use.
+
+This attribute also has a custom warning code `XAOBS001` so that it can be suppressed
+independently of "normal" obsolete API.
+
+Support for this property was added in .NET 8.
+
+This property is `True` by default.
```

Comment:
Should we also add:

```markdown
If you would instead prefer to *remove* members that have the `@RestrictTo` annotation *or* are in non-exported Java packages, you can use [Transform files](https://learn.microsoft.com/xamarin/android/platform/binding-java-library/customizing-bindings/java-bindings-metadata#metadataxml-transform-file) to prevent these types from being bound:

            <attr path="//class[@annotated-visibility]"
                name="visibility">kotlin-internal</attr>
            <attr path="//interface[@annotated-visibility]"
                name="visibility">kotlin-internal</attr>

```

Reply:
…formatted properly to use XML block formatting, as appropriate.